manifests/ca.pp

```diff
@@ -104,6 +107,7 @@
   $key_name     = '',
   $key_ou       = '',
   $tls_auth     = false,
+  $easyrsa_source = $::openvpn::params::easyrsa_source,
```

Comment:
Instead of hard coding this, I please handle this in `params.pp`. Or what's the reason to handle this here?